roles/lib_openshift_api/build/generate.py

```diff
@@ -27,18 +28,23 @@
                           'src/secret.py',
                           'ansible/secret.py',
                          ],
+         'oc_edit.py': ['src/base.py',
+                        '../../lib_yaml_editor/build/src/yedit.py',
+                        'src/edit.py',
+                        'ansible/edit.py',
+                       ],
         }
 
 
 def main():
     ''' combine the necessary files to create the ansible module '''
-    openshift_ansible = ('../library/')
+    library = os.path.join(OPENSHIFT_ANSIBLE_PATH, '../library/')
```

Comment:
Are you sure this is the right way to do this?

I thought it should be something more like this (so that the library puts the path together for us):
```
library = os.path.join(OPENSHIFT_ANSIBLE_PATH, '..', 'library')
```